tests/workbenches/notebook-controller/test_spawning.py

```diff
@@ -0,0 +1,201 @@
+import ocp_resources.persistent_volume_claim
+import ocp_resources.route
+import ocp_resources.resource
+
+import kubernetes.client
+from kubernetes.dynamic import DynamicClient
+
+from tests.workbenches.conftest import OdhAnnotationsLabels, OdhConstants, PodUtils
+from tests.workbenches.docs import TestDoc, SuiteDoc, Contact, Desc, Step
+
+
+# @allure.title("Test Authentication")
+# @allure.description("Verifies deployments of Notebooks via GitOps approach")
+# @allure.tag("NewUI", "Essentials", "Authentication")
+# @allure.severity(allure.severity_level.CRITICAL)
+# @allure.label("owner", "John Doe")
+# @allure.link("https://dev.example.com/", name="Website")
+# @allure.issue("AUTH-123")
+# @allure.testcase("TMS-456")
+
+
+@SuiteDoc(
+    description=Desc("Verifies deployments of Notebooks via GitOps approach"),
+    beforeTestSteps={
+        Step(value="Deploy Pipelines Operator", expected="Pipelines operator is available on the cluster"),
+        Step(value="Deploy ServiceMesh Operator", expected="ServiceMesh operator is available on the cluster"),
+        Step(value="Deploy Serverless Operator", expected="Serverless operator is available on the cluster"),
+        Step(value="Install ODH operator", expected="Operator is up and running and is able to serve it's operands"),
+        Step(value="Deploy DSCI", expected="DSCI is created and ready"),
+        Step(value="Deploy DSC", expected="DSC is created and ready"),
+    },
+    afterTestSteps={
+        Step(
+            value="Delete ODH operator and all created resources",
+            expected="Operator is removed and all other resources as well",
+        )
+    },
+)
+class TestNotebookST:
+    @classmethod
+    @property
+    @functools.cache
+    def logger(cls):
+        return logging.getLogger(cls.__name__)
+
+    DS_PROJECT_NAME: str = "test-notebooks"
+
+    NTB_NAME: str = "test-odh-notebook"
+    NTB_NAMESPACE: str = "test-odh-notebook"
+
+    @TestDoc(
+        description=Desc("Create simple Notebook with all needed resources and see if Operator creates it properly"),
+        contact=Contact(name="Jakub Stejskal", email="jstejska@redhat.com"),
+        steps={
+            Step(
+                value="Create namespace for Notebook resources with proper name, labels and annotations",
+                expected="Namespace is created",
+            ),
+            Step(value="Create PVC with proper labels and data for Notebook", expected="PVC is created"),
+            Step(
+                value="Create Notebook resource with Jupyter Minimal image in pre-defined namespace",
+                expected="Notebook resource is created",
+            ),
+            Step(
+                value="Wait for Notebook pods readiness",
+                expected="Notebook pods are up and running, Notebook is in ready state",
+            ),
+        },
+    )
+    def testCreateSimpleNotebook(self, function_resource_manager, admin_client, unprivileged_client):
```

Comment:
all setup steps (e.g namespace, pvc) should be done in fixtures
please check existing fixtures and re-use

Reply:
done